Modules/ITS/src/runITS.cxx

```diff
@@ -0,0 +1,116 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   runITS.cxx
+/// \author Zhaozhong Shi
+///
+/// \brief This is an executable showing QC Task's usage in Data Processing Layer.
+///
+/// This is an executable showing QC Task's usage in Data Processing Layer. The workflow consists of data producer,
+/// which generates arrays of random size and content. Its output is dispatched to QC task using Data Sampling
+/// infrastructure. QC Task runs exemplary user code located in SkeletonDPL. The checker performes a simple check of
+/// the histogram shape and colorizes it. The resulting histogram contents are shown in logs by printer.
+///
+/// QC task and Checker are instantiated by respectively TaskFactory and CheckerFactory,
+/// which use preinstalled config file, that can be found in
+/// ${QUALITYCONTROL_ROOT}/etc/qcTaskDplConfig.json or Framework/qcTaskDplConfig.json (original one).
+///
+/// To launch it, build the project, load the environment and run the executable:
+///   \code{.sh}
+///   > aliBuild build QualityControl --defaults o2
+///   > alienv enter QualityControl/latest
+///   > runTaskDPL
```

Comment:
Please update this piece of documentation or delete it.

Reply:
It's done.